src/operators/terminology_resolver.cpp

```diff
@@ -0,0 +1,180 @@
+
+#include "datatypes/pointcollection.h"
+#include "datatypes/linecollection.h"
+#include "datatypes/polygoncollection.h"
+
+#include "operators/operator.h"
+
+#include <json/json.h>
+#include "../util/terminology.h"
+
+/**
+ * Operator for resolving strings with the terminology service from gfbio: https://terminologies.gfbio.org/
+ *
+ * parameters:
+ *  - column: name of the textual column to resolve
+ *  - terminology: the terminology used to resolve
+ *  - key: the json field of the result to be saved in the new column. "label" if not provided.
+ *         if requested field is an array, the first element will be returned.
+ *  - match_type: "exact", "included", "regex", see TerminologyService search API. "exact" if not provided.
+ *  - first_hit: bool, see TerminologyService search API. true if not provided.
+ *  - name_appendix: name of the next column for the resolved text values
```

Comment:
> name of the *next* column for the resolved text values
should say *new* column I guess.

why is it called *appendix* if it can also replace the entire string? With this scheme it's also not possible to name the new column "teminology" ;) I'd propose calling the option `resolved_column` (or `resolved_attribute` depending on the issue above) and just accepting a custom string. the user can put the terminology name there if he wants to.

Reply:
we can also handle the appendix part in the UI. like everything...